_layouts/podcast.html

```diff
@@ -0,0 +1,25 @@
+---
+layout: default
+---
+{% assign author = site.data.authors[page.author] %}
+<div class="post-container">
+  <div class="post-head">
+    <a href="/podcast.html" class="post-back-button">Back to all podcasts</a>
+    <h1>{{ page.title }}</h1>
+  </div>
+  <div class="post-row">
+    <div class="post-content">
+      <em>Posted on {{ page.date | date: "%-d %B %Y" }}</em>
+      {{ content }}
+
+      Brink supports and mentors new contributors to open source Bitcoin development through our fellowship program, and supports the work of established Bitcoin protocol engineers through our grants program. <a href="https://brink.dev/programs">Learn more</a> about our efforts.
```

Comment:
Can we some kind of page break between the content and the Brink description? Otherwise, they run together a bit:

![image](https://user-images.githubusercontent.com/1063656/140376089-057b9d52-0e1b-4a16-b10e-7b4446251dea.png)

Reply:
I think that summary makes more sense to have on the podcatcher summaries and not so much on our website so Im going to remove it.

Reply:
Oh, I like having the show notes on our own page. If we ever change the service we use for hosting our podcasts, then we'll still have notes locally.